src/index.ts

```diff
@@ -83,21 +83,22 @@ const getAssetFromKV = async (event: FetchEvent, options?: Partial<Options>): Pr
   const request = event.request
   const ASSET_NAMESPACE = options.ASSET_NAMESPACE
   const ASSET_MANIFEST = options.ASSET_MANIFEST
+    ? JSON.parse(options.ASSET_MANIFEST)
```

Comment:
Should we always assume that `ASSET_MANIFEST` is json, and error if not? I don't think we have any case where `ASSET_MANIFEST` is anything but json...

Reply:
Thanks for pointing this out. `ASSET_MANIFEST` is actually a text blob. Currently if someone really wanted to not have an asset manifest that is OK it just won't cache and they will use a key without the hash. I set this so it will only run  `JSON.parse` is a text blob is passed in. I added a commit so the types were more clear